src/test/js/edgeCases/folder.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
             const freestylePage = browser.page.pipeline()
                 .forUrl(response.value, jobName);
 
+            // FIXME: Talk to Tom, why next lines are not working as expected
             // freestylePage.buildStarted(function () {
```

Comment:
Are you on the right page object? Sounds like `browser.page.pipeline()` would be a `pipeline` page, yet you are calling it `freestylePage`?